src/components/UserSelector.tsx

```diff
@@ -1,24 +1,31 @@
-import React, { useContext, useEffect, useState } from 'react';
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect } from 'react';
 import classNames from 'classnames';
-import { UserContext } from './UsersContext';
 import { User } from '../types/User';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import { getUsers } from '../features/users/users';
+import * as usersActions from '../features/users/usersSlice';
+import * as authorActions from '../features/author/authorSlice';
 
 type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.